frontend/src/pages/destructionlist/DestructionListCreate.tsx

```diff
@@ -233,79 +238,105 @@ export function DestructionListCreatePage({
   /**
    * Get called when the selection is submitted.
    */
-  const onCreate = async () => {
+  const onSubmitSelection = () => setModalOpenState(true);
+
+  /**
+   * Gets called when the form is submitted.
+   */
+  const onSubmitForm = async (event: FormEvent, data: SerializedFormData) => {
     const zaakSelection = await getZaakSelection(DESTRUCTION_LIST_CREATE_KEY);
     const zaakUrls = Object.entries(zaakSelection)
       .filter(([, selected]) => selected)
       .map(([url]) => url);
-    const assigneeIds = [1]; // TODO: Add a modal with actual assignees
+    const { name, assigneeIds } = data;
 
-    const data = new FormData();
-    data.append("name", nameState);
-    zaakUrls.forEach((url) => data.append("zaakUrls", url));
-    assigneeIds.forEach((id) => data.append("assigneeIds", String(id)));
+    const formData = new FormData();
+    formData.append("name", name as string);
+    zaakUrls.forEach((url) => formData.append("zaakUrls", url));
+    (assigneeIds as string[])
+      .filter((id) => id)
+      .forEach((id) => formData.append("assigneeIds", String(id)));
 
-    submit(data, { method: "POST" });
+    submit(formData, { method: "POST" });
+    setModalOpenState(false);
```

Comment:
In the case that the creation of the list fails, this will still close the modal without showing feedback to the record manager?